api/client/webclient/webconfig.go

```diff
@@ -26,19 +26,19 @@ const (
 	WebConfigAuthProviderOIDCType = "oidc"
 	// WebConfigAuthProviderOIDCURL is OIDC webapi endpoint.
 	// redirect_url MUST be the last query param, see the comment in parseSSORequestParams for an explanation.
```

Comment:
These comments are now a bit confusing as they don't exist in the path any longer. Maybe expand this to include a note about them being added by the UI.